CHANGELOG.md

```diff
@@ -84,6 +84,8 @@
   - fixed incorrect data display in service company subscriptions [#1191](https://github.com/eclipse-tractusx/portal-frontend/pull/1191)
   - fixed error message scenario post technical user deletion action [#1164](https://github.com/eclipse-tractusx/portal-frontend/pull/1164)
   - fixed duplicate api records on load [#1206] (https://github.com/eclipse-tractusx/portal-frontend/pull/1206)
+- **Service Marketplace**
+  - Filter service in Service Marketplace wasn't filtering for DE language [#1209](https://github.com/eclipse-tractusx/portal-frontend/pull/1209)
```

Comment:
Hi @Usmanfee  we have an continuous issue in the changelog file, the entry is at the wrong place and it's not even shown as conflict. I suggest you revert any change to the changelog and add the information which should be added to the changelog in the PR description and I add it then at the next release to the changelog file, sorry for the inconvenience.